app/domain/postalCodeCoordinates.ts

```diff
@@ -0,0 +1,15 @@
+import fs from 'fs';
+
+const POSTNUMMERE = JSON.parse(
+  fs.readFileSync(`${__dirname}/postnummere.json`, 'utf8')
+);
+
+export default function(postalCode: string): number[] {
+  const r = POSTNUMMERE.find((e: any) => e.postnummer === postalCode);
```

Comment:
Ikke så viktig å fikse, men greit å alltid skrive variabelnavn helt ut. Da er det litt enklere å skjønne hva det er for noe (litt vanskelig å si hva `r`er nå)

Reply:
Helt enig, derfor det er fint med reviews :)